### PR TITLE
[error-monitoring] Delete _cron/monitor endpoint

### PR DESCRIPTION
**Pull request by estherkim**
Also delete associated functions.

Set `error-reporting` as default ISSUE_REPO_NAME value

### CODE REVIEW

**Review by rcebulko — COMMENTED** [review]
Alternate idea: let's just enable this again? We originally turned it off because we thought it was creating tons of extra issues for some reason, but it turns out that reason was a zombie project totally separate. Since that's been resolved, we can probably return to auto-filing issues each morning (and they auto-tag the release on-duty engineers). We used to have a `cron.yaml` config like:


```yaml
cron:
- description: "File issues for errors in production"
  url: /_cron/monitor?serviceType=production
  schedule: every day at 7:00am
- description: "File issues for errors in 1%/opt-in"
  url: /_cron/monitor?serviceType=development
  schedule: every day at 7:02am
- description: "File issues for errors in experiments"
  url: /_cron/monitor?serviceType=experiments
  schedule: every day at 7:04am
- description: "File issues for errors in nightly"
  url: /_cron/monitor?serviceType=nightly
  schedule: every day at 7:06am
```

**Comment by estherkim** [comment]
Oh, didn't you say it was turned off due to privacy reasons?

**Comment by estherkim** [comment]
Hmm, so delete?

**Comment by rcebulko** [comment]
> Hmm, so delete?

Maybe? I don't have strong feelings here

**Comment by estherkim** [comment]
I'm all for deleting. Can I get a PR review?

**Review by rcebulko — APPROVED** [review]

File: error-monitoring/index.ts [hunk 1]
```diff
@@ -37,9 +33,12 @@ import express from 'express';
...
 
-const GITHUB_REPO = process.env.GITHUB_REPO || 'ampproject/amphtml';
-const [GITHUB_REPO_OWNER, GITHUB_REPO_NAME] = GITHUB_REPO.split('/');
-const ISSUE_REPO_NAME = process.env.ISSUE_REPO_NAME || GITHUB_REPO_NAME;
+const ERROR_ISSUE_ENDPOINT =
+  process.env.ERROR_ISSUE_ENDPOINT ||
+  'https://amp-error-monitoring.uc.r.appspot.com/error-issue';
+const GITHUB_SLUG = process.env.GITHUB_SLUG || 'ampproject/amphtml';
+const [GITHUB_REPO_OWNER, GITHUB_REPO_NAME] = GITHUB_SLUG.split('/');
+const ISSUE_REPO_NAME = process.env.ISSUE_REPO_NAME || 'error-reporting';
```
Comment (rcebulko) [inline error-monitoring/index.ts:41]: As a default I think `GITHUB_REPO_NAME` is more sensible here actually

**Pull request merged**